examples/diffusion/python_stable_diffusion_21/gradio_reqirements.txt

```diff
@@ -1,7 +1,7 @@
 #####################################################################################
```

Comment:
The file is spelt incorrectly.  You are missing the "u" in gradio_requirements.txt